From 4d2336646a28781271f1c199db7158542dcad449 Mon Sep 17 00:00:00 2001
From: Re4son <3520622-re4son@users.noreply.gitlab.com>
Date: Mon, 28 Dec 2020 11:39:05 +1100
Subject: [PATCH] ts.tv_nsec

Signed-off-by: Re4son <3520622-re4son@users.noreply.gitlab.com>
---
 coreutils/date.c | 12 +++++++-----
 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git a/coreutils/date.c b/coreutils/date.c
index b9b7fd2cb..f77ca8088 100644
--- a/coreutils/date.c
+++ b/coreutils/date.c
@@ -263,11 +263,13 @@ int date_main(int argc UNUSED_PARAM, char **argv)
 		xstat(filename, &statbuf);
 		ts.tv_sec = statbuf.st_mtime;
 #if ENABLE_FEATURE_DATE_NANO
-		ts.tv_nsec = statbuf.st_mtim.tv_nsec;
-		/* Some toolchains use .st_mtimensec instead of st_mtim.tv_nsec.
-		 * If you need #define _SVID_SOURCE 1 to enable st_mtim.tv_nsec,
-		 * drop a mail to project mailing list please
-		 */
+        #if !defined(ANDROID) && !defined(__ANDROID__)
+            ts.tv_nsec = statbuf.st_mtim.tv_nsec;
+            /* Some toolchains use .st_mtimensec instead of st_mtim.tv_nsec.
+             * If you need #define _SVID_SOURCE 1 to enable st_mtim.tv_nsec,
+             * drop a mail to project mailing list please
+             */
+        #endif
 #endif
 	} else {
 #if ENABLE_FEATURE_DATE_NANO
-- 
2.29.2

